smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/serde/member/MemberShapeDecodeGenerator.kt

```diff
@@ -219,7 +219,7 @@ open class MemberShapeDecodeGenerator(
             }
             // Provide a default value dependent on the type.
             return when (targetShape) {
-                is EnumShape -> " ?? .${enumDefaultValue(targetShape, it.expectStringNode().value)}"
+                is EnumShape -> " ?? ${enumDefaultValue(writer, targetShape, it.expectStringNode().value)}"
```

Comment:
`writer` is passed in so the symbol in the default value can be tracked for import